_sass/docs.scss

```
@@ -179,4 +179,26 @@
       }
     }
   }
+
+  figure {
+    img {
+      border-radius: 3px;
+      background-color: $sr-white;
+      border: 1px solid darken($sr-light-grey, 10%);
+      padding: 3px;
+    }
+
+    &.spaced-figure-non-bold-caption {
+      figcaption {
+        font-weight: normal;
+        font-size: 1em;
+      }
+
+      margin-bottom: 1.5rem;
+
+      &:last-of-type {
+        margin-bottom: 0;
+      }
+    }
+  }
```

To make the figures work I think we need to more completely copy the CSS from the website:
```suggestion
  figure {
    img {
      width: 100%;
      border-radius: 3px;
      background-color: $sr-white;
      border: 1px solid darken($sr-light-grey, 10%);
      padding: 3px;
    }

    &.spaced-figure-non-bold-caption {
      figcaption {
        font-weight: normal;
        font-size: 1em;
      }

      margin-bottom: 1.5rem;
      &:last-of-type {
        margin-bottom: 0;
      }
    }
  }

  figcaption {
    font-size: 0.8em;
    font-weight: bold;
    text-align: center;
  }
```
The changes here are: the image at 100% width and the standalone `figcaption` block.